docs/reference/examples.md

```diff
@@ -28,9 +28,7 @@ using modern Javascript, but `await` calls can also be rendered with promises.
 // That site exposes a global StellarSdk object you can use.
 // To run this code in the Chrome, open the console tab in the DevTools.
 // The hotkey to open the DevTools console is Ctrl+Shift+J or (Cmd+Opt+J on Mac).
-
-// To use in node, do `npm install stellar-sdk` and uncomment the following line.
-// const StellarSdk = require('stellar-sdk');
+const StellarSdk = require('stellar-sdk');
```

Comment:
Only thing about changing this is that it doesn't work by copy-pasting into devtools console anymore. Might not be worth changing since that's what the comment above explains how to do.

Reply:
Devtools console isn't going to work anyway because `StellarSdk` isn't present on the page.

Reply:
(It probably should be though.)